internal/plan/plan.go

```diff
@@ -149,6 +151,47 @@ func (a *Action) DetailedString() string {
 	}
 }
 
+// Apply commits all required changes for action to given Host.
+func (a *Action) Apply(ctx context.Context, host hostPkg.Host) error {
+	args := []any{}
+	diffStr := a.DiffString()
+	if len(diffStr) > 0 {
+		args = append(args, []any{"diff", diffStr}...)
+	}
+	ctx, _ = log.MustContextLoggerSection(ctx, a.String(), args...)
+
+	if len(a.ApplyResources) == 0 {
+		return nil
+	}
+
+	isGroupResource := resourcesPkg.IsGroupResource(
+		resourcesPkg.GetResourceTypeName(a.ApplyResources[0]),
```

Comment:
If `ApplyResources` empty ?

Reply:
I believe the check a few lines above protects against that, did I miss something?

Reply:
No, I was the one who missed, :)